client/cmd/dexc/main.go

```diff
@@ -56,11 +56,16 @@ func main() {
 	logStdout := func(msg []byte) {
 		os.Stdout.Write(msg)
 	}
-	logMaker := ui.InitLogging(logStdout, cfg.DebugLevel, !cfg.LocalLogs)
+	utc := !cfg.LocalLogs
+	if cfg.Net == dex.Simnet {
+		utc = false
+	}
```

Comment:
Oh. I really just meant for the dcrdex simnet harness `dex/testing/dcrdex/harness.sh`, but this is fine too.